docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -4,17 +4,17 @@ title: Migrating from v1 to v2
 
 Looking for the v1 docs? [Find them here](https://v1.gatsbyjs.org/).
 
-> This document is a work in progress. Have you upgraded your site and run into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
+> This document is a work in progress. Have you upgraded your site and ran into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
```

Comment:
I don't think _ran_ is right here? Maybe we could shorten the sentence though:

```suggestion
> This document is a work in progress. Have you run into something that's not covered here? [Add your changes on GitHub](https://github.com/gatsbyjs/gatsby/edit/master/docs/docs/migrating-from-v1-to-v2.md)!
```

Reply:
I was a little confused with this sentence. Thank you!